assertions/src/main/java/io/camunda/zeebe/process/test/assertions/FormAssert.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright © 2021 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.camunda.zeebe.process.test.assertions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.api.response.Form;
+import io.camunda.zeebe.process.test.filters.RecordStream;
+import org.assertj.core.api.AbstractAssert;
+
+/**
+ * Assertions for {@link Form} instances.
+ *
+ * <p>These asserts can be obtained via:
+ *
+ * <pre>{@code
+ * final DeploymentEvent deploymentEvent =
+ *         client.newDeployCommand().addResourceFile(file).send().join();
+ *
+ * final FormAssert formAssert =
+ *         assertThat(deploymentEvent)
+ *             .extractingProcessByBpmnProcessId(PROCESS_ID);
+ * }</pre>
+ */
+public class FormAssert extends AbstractAssert<FormAssert, Form> {
+
+  private final RecordStream recordStream;
```

Comment:
🔧 This one is unused. It would be good if we add `isCreated` method to assert whether form created record is present in the record stream. Therefore, this field will have an effect as well. Also, we need to add tests for the form created case.

Reply:
Thanks for the input. I changed a bit the implementation, I added a new class for correctly inspect the log stream